### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by vishnu-chiluveri**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,10 +1,5 @@
 <div class="hero">
+
```
Comment (Chandukasireddy) [inline index.html:2]: add another line

File: index.html [hunk 2]
```diff
@@ -1,10 +1,5 @@
 <div class="hero">
```
Comment (Chandukasireddy) [inline index.html:1]: another line

File: index.html (hunk 1 above)
Comment (vishnu-chiluveri) [reply index.html:2]: problem resolved

**Review by vishnu-chiluveri — COMMENTED** [review]
problem solved

**Pull request merged**